packages/shared/src/components/profile/ProfileTooltipContent.tsx

```diff
@@ -42,11 +42,11 @@ export function ProfileTooltipContent({
           {user.bio}
         </p>
       )}
-      {tags?.length && (
+      {tags?.length ? (
```

Comment:
Why do we need the hardcoded null return?

Reply:
It displays the string `0` when the length is 0 instead of conditionally rendering. If we turn it into a boolean, it might return the word `false`.